### PR TITLE
[Account][Order] Be able to pay for unpaid orders

### PR DESCRIPTION
**Pull request by Arminek**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | no |
| BC breaks?      | no |
| License         | MIT |

### CODE REVIEW

File: src/Sylius/Behat/Context/Ui/Shop/AccountContext.php [hunk 1]
```diff
@@ -468,6 +468,17 @@ public function iShouldSeeAsProvinceInTheBillingAddress($provinceName)
     }
 
     /**
+     * @Then /^I should be able to change payment method for (this order)$/
+     */
+    public function iShouldBeAbleToChangePaymentMethodForThisOrder(OrderInterface $order)
+    {
+        Assert::true(
+            $this->orderIndexPage->beAbleToChangePaymentMethodForOrder($order),
+            sprintf('I should be able to change payment method for %s', $order->getNumber())
```
Comment (Zales0123) [inline src/Sylius/Behat/Context/Ui/Shop/AccountContext.php]: Missing dot at the end of sentence :(

File: src/Sylius/Behat/Page/Shop/Account/Order/IndexPageInterface.php [hunk 2]
```diff
@@ -29,4 +30,9 @@ public function countOrders();
      * @return bool
      */
     public function isOrderWithNumberInTheList($number);
+
+    /**
+     * @param OrderInterface $order
+     */
+    public function beAbleToChangePaymentMethodForOrder(OrderInterface $order);
```
Comment (Zales0123) [inline src/Sylius/Behat/Page/Shop/Account/Order/IndexPageInterface.php]: *Primo*: missing ``@return bool``
*Secundo*: what would you say for a more... *bool-returning* method name, like ``isItPossibleToChangePaymentMethodForOrder(...)`` or sth like this?

File: src/Sylius/Bundle/ShopBundle/Resources/views/Grid/Action/pay_for_order.html.twig [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{% import 'SyliusUiBundle:Macro:buttons.html.twig' as buttons %}
+
+{% if data.paymentState in ['awaiting_payment'] %}
+
```
Comment (Zales0123) [inline src/Sylius/Bundle/ShopBundle/Resources/views/Grid/Action/pay_for_order.html.twig]: Is this blank line needed?

File: src/Sylius/Behat/Context/Ui/Shop/AccountContext.php (hunk 1 above)
Comment (pamil) [inline src/Sylius/Behat/Context/Ui/Shop/AccountContext.php]: `for %s` -> `for order %s`

**Comment by pjedrzejewski** [comment]
Thanks Arek!

**Pull request merged**